main.nf

```diff
@@ -43,6 +44,7 @@ def helpMessage() {
 	                                create overlapping file names that crash MultiQC.
 	  --split [str]                 A string that will be used between the prepended run/folder name and the sample name. (default: "-")
 	                                May not be present in run/folder names and no underscore(s) allowed. Only used with "--multipleSequencingRuns"
+	  --pacbio			If PacBio data. Use this option together with --single_end and --manifest.
```

Comment:
Well, it was still present in the main.nf and some other places a couple of weeks ago, so I thought I should use it... It is not really needed as it is now, the pacbio option can be used to determine how to read input data, but it might be useful if other types of data is supported in the future? I have no strong opinions here.

Reply:
Are some of the processes that are executed now with `params.single_end = true` also usable for single end Illumina data? If yes, than future additions (that I do not plan at this time) might profit. But I think in this case `single_end` should be set to true without specifying it. I dislike that currently `--single_end` and `--pacbio` need to be specified simultaneously. `--pacbio` defines that data is single end, there is no paired end pacbio data, is there?
If processes executed now with `params.single_end = true` are not potentially usable for single end Illumina data, I think it would be better to remove `--single_end` and use only `--pacbio`.

Reply:
I think the single end option is potentially usable for other types of single end data. However, it's a god idea to set it without specifying it for PacBio data. Thanks for the suggestion.